### DISCUSSION TITLE
Error in the X-Z Direction of Pole Figure

### DISCUSSION THREAD

**Discussion by rockdef** [mtex-toolbox/mtex#1974] [Ask Anything]
Hi MTEX Community, 

While plotting the Pole Figure (one point per grain), I encountered a problem and wasn't able to fix it. 

**Problem**: Why is the X-axis plotted in the N-S direction instead of my preferred E-W direction in all cases except for (110)? 
```
%% Specify Crystal and Specimen Symmetries
CS = {... 
  'notIndexed',...
  crystalSymmetry('12/m1', [9.9 18 5.3], [90,105.2,90]*degree, 'X||a*', 'Y||b*', 'Z||c', 'mineral', 'Hornblende', 'color', [1 0.6 0])};

%% Miller Indices
h_hbl = [Miller(1,0,0,CS{2},'hkl'), Miller(0,1,0,CS{2},'hkl'),Miller(0,0,1,CS{2},'hkl'), Miller(1,1,0,CS{2},'hkl')];

%% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outOfPlane');

%% Import the Data
ebsd = EBSD.load(fname,CS,'interface','crc',...
  'convertEuler2SpatialReferenceFrame','setting 2');

%% Rotation 
ebsd = rotate(ebsd,rotation('axis',vector3d.X,'angle',180*degree));

%% Compute preliminary grain boundaries
[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd,'angle',[10 1]*degree,'Boundary','tight');

%% Plotting CPO (one point per grain) 
% We should probably derive both from an odf which you can calculate from either all orientations (one way) 
% or from grains.meanOrientation (another way). In both cases, guessing a kernel using calcKernel() might not be a bad idea.

% FIRST WAY 
% psi = calcKernel(grains('Hornblende').meanOrientation); 
% odf_1ppg = calcDensity(ebsd('Hornblende').orientations,'kernel',psi);
odf_1ppg = calcDensity(ebsd('Hornblende').orientations,'halfwidth',10*degree);

% Pole figure 
figure 
plotPDF(odf_1ppg,h_hbl,'antipodal','MarkerSize',2,'micronBar','off','figSize','medium')
annotation('textbox','String',ngrains,'FitBoxToText','on','lineStyle','none');
setColorRange([0 10]);
cb = mtexColorbar('Title','M.U.D.- 1 Point per grain');
mtexColorMap(WhiteJetColorMap)
set(gca,'Color',[.5 .5 .5])
```
Thanks in advance! 
![POLE FIGURE](https://github.com/mtex-toolbox/mtex/assets/130278695/50866ed1-42a4-444a-9e51-4ee1f9bc4ce3)

**Comment by kilir** [comment]
Hi,
that looks indeed strange. What version of mtex are you using?  If 6.0.beta.x, have a look at the change log, setting plotting conventions has changed.
Your annotation comes from the default in `mtex_settings`? 

Cheers,
Rüdiger

**Reply by rockdef** [reply]
Hi Rüdiger, 
Thanks for the reply. 
Yes, I am using 6.0.beta2 (latest version). 

Earlier it was fine in the previous version, I believe it is coming from default mtex_settings. 
I am not sure how to change it. Do you have any suggestions? 

Thanks in advance!

**Reply by kilir** [reply]
Have a look in the change log: https://mtex-toolbox.github.io/changelog#1

**Reply by rockdef** [reply]
Ok. Thank you for the suggestion. 
I checked with the previous version and it works finely. 
So surely there is a problem with MTEX's latest version as you said.

**Comment by inandi2** [comment]
Hi,
I am having similar issues with the rotation of the axis and the pole figure. I want my Y-axis to appear on the south and X-axis on the east (similar to the bottom figure). I had it worked out on the previous MTEX version but version 6 seems to be not working. Any comments will be highly appreciated. 

Thanks in advance!

![image](https://github.com/mtex-toolbox/mtex/assets/61553917/238831a5-5445-495e-955b-b0c9015434b1)
